app/components/slack-context-menu.js

```diff
@@ -138,7 +149,7 @@
 	}
 
 	module.exports = {
-		handleRightClick: function (window, target, x, y) {
+		handleRightClick: function (window, target, x, y, teamUrl) {
```

Comment:
Then in the later invocations, we can always have the syntax `build*Menu(menu, params);`